locales/fr_FR/pcsx2_Main.po

```diff
@@ -3009,227 +3014,4 @@ msgid ""
-#~ "version du logiciel."
-
-#~ msgid "Always on Top"
-#~ msgstr "Toujours visible"
-
-#~ msgid ""
-#~ "When checked the log window will be visible over other foreground windows."
-#~ msgstr ""
-#~ "Une fois cochée, la fenêtre de dialogue restera toujours par-dessus les "
-#~ "autres fenêtres."
-
-#~ msgid "&Iso"
-#~ msgstr "ISO"
-
-#~ msgid "Reboot CDVD (f&ull)"
-#~ msgstr "Relancer le CDVD (complet)"
-
-#~ msgid "Hard reset of the active VM."
-#~ msgstr "Redémarrage intégral du système d'émulation."
-
-#~ msgid "Reboot CDVD (&fast)"
-#~ msgstr "Relancer le CDVD (rapide)"
-
-#~ msgid "Reboot using fast BOOT (skips splash screens)"
-#~ msgstr ""
-#~ "Redémarrage rapide du système d'émulation (saute les logos PS2/Sony)."
-
-#~ msgid "Ignore Bus Direction on Path3 Transfer - Used for Hotwheels"
-#~ msgstr ""
-#~ "Ignorer la direction du port sur le transfert Path3 - Utile pour le jeu "
-#~ "Hot Wheels"
-
-#~ msgid ""
-#~ "Resets all speedhack options to their defaults, which consequently turns "
-#~ "them all OFF."
-#~ msgstr ""
-#~ "Réinitialise les options des speedhacks à leurs valeurs par défaut (à "
-#~ "noter que cette opération les désactivera tous)."
-
-#~ msgid "Betatesting"
-#~ msgstr "Beta-test"
-
-#~ msgid "BIOS Selector"
-#~ msgstr "Choix du BIOS"
-
-#~ msgid "Select CDVD source iso..."
-#~ msgstr "Choisir un ISO..."
-
-#~ msgid "PCSX2 - SSE2 Recommended"
-#~ msgstr "PCSX2 - SSE2 Recommandé"
-
-#~ msgid "Dynamically toggle Vsync depending on frame rate (read tooltip!)"
-#~ msgstr ""
-#~ "Bascule Vsync dynamique basée sur la frame rate (lisez le tooltip !)"
-
-#, fuzzy
-#~ msgid "%s  %d.%d.%d"
-#~ msgstr "%s  %d.%d.%d %s"
-
-#, fuzzy
-#~ msgid "%s  %d.%d.%d-%lld%s (git)  %s"
-#~ msgstr "%s  %d.%d.%d.%d%s (svn)  %s"
-
-#~ msgid ""
-#~ "The safest way to make sure that all game fixes are completely disabled."
-#~ msgstr ""
-#~ "La meilleure méthode pour s'assurer que tous les patchs ont été "
-#~ "désactivés."
-
-#~ msgid "(modded)"
-#~ msgstr "(modded)"
-
-#~ msgid ""
-#~ "Cannot load this savestate.  The state is from an incompatible edition of "
-#~ "PCSX2 that is either newer than this version, or is no longer supported."
-#~ msgstr ""
-#~ "Impossible de charger cette sauvegarde. L'emplacement provient d'une "
-#~ "version de PCSX2 incompatible (trop ancienne, ou nouvelle)."
-
-#~ msgid ""
-#~ "Cannot load this savestate. The state is an unsupported version, likely "
-#~ "created by a newer edition of PCSX2."
-#~ msgstr ""
-#~ "Impossible de charger cette sauvegarde. L'emplacement n'est pas supporté "
-#~ "car il a sûrement été créé à partir d'une version de PCSX2 non-gérée."
-
-#~ msgid "mVU Block Hack"
-#~ msgstr "mVU Block Hack"
-
-#~ msgid ""
-#~ "Good Speedup and High Compatibility; may cause bad graphics, SPS, etc..."
-#~ msgstr ""
-#~ "Amélioration sensible des performances et compatibilité élevée, mais "
-#~ "pourrait causer quelques bugs graphiques, SPS, etc..."
+"extensions SSE2."
```

Comment:
Same here Newline breaking. Guess ill have to test to make sure it doesnt break.

Reply:
It normally shouldn't, I suppose that it might put it back if you open the PO file...

Reply:
I'll wait till appveyor is done and test myself and if it works i'll blame GitHub. 😉